awx/main/tasks.py

```diff
@@ -1186,16 +1186,19 @@ def pre_run_hook(self, instance, private_data_dir):
         '''
         Hook for any steps to run before the job/task starts
         '''
+        logger_job_lifecycle.info(f"{instance._meta.model_name}-{instance.id} pre run", extra={'type': instance._meta.model_name, 'template_name': instance.unified_job_template.name, 'job_id': instance.id, 'state': 'pre_run'})
```

Comment:
I like the use of the hooks for this 👍

Reply:
Instead of `'job_id'` could we use `self.event_data_key` for accuracy?

Reply:
thinking about parsing this json -- would we want a fixed key here across all entries? maybe it could be named `task_id` to be more generic

Reply:
Yea, that's a good point.  If we want to stick with `job_id`, that probably makes sense considering I think there's additional context in here that tells you what *type* of job it is.